src/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+interface TypographyProps {
+  size: number;
+  label: string;
+}
```

Comment:
1.  can you move all types to a separate file f.e. typography.types.d.ts
2. can yo add [jsdoc](https://jsdoc.app/about-getting-started.html) comments for all properties
3. can you rename interface to ITypographyProps and follow this naming convention for interface names

Reply:
https://github.com/igorizviekov/mt-booking_app/commit/9890b68f16912f4f8fe1b42285ed492a6e78b23c
https://github.com/igorizviekov/mt-booking_app/commit/b0640ecb59e66c95da4a773d7359b4cdb0096d71